GameData/ROEngines/PartConfigs/PioneerVernier.cfg

```diff
@@ -0,0 +1,207 @@
+//  Dry Mass: 1 lb // - Approximate weight of 120ml solid fuel.  Volume from cylinder size.
+//  Thrust: 13 lb  // Total impulse based on propellant load and relealized delta-v on Pioneer 1 mission
+//  Burn Time: 16s //
+//  ISP: 235
+//
+//  1958 NASA/USAF Space Probes (ABLE-1): https://ntrs.nasa.gov/citations/19990008156
+// 
+//	=================================================
+
+
+PART
+{
+	author = itsRyan
+	name = PioneerVernier
+	module = Part
+
+	RSSROConfig = True
+
+	MODEL
+	{
+		model = ROEngines/Assets/ItsRyan/PioneerVernier
+		scale = 1.0, 1.0, 1.0
+	}
+
+	rescaleFactor = 1.0
+	scale = 1.0
+
+	CoMOffset = 0.0, 1.1, 0.0
+
+	node_stack_top = 0.0,0.108,0.0,0.0,1.0,0.0,0
+	node_stack_bottom = 0.0,0.0,0.0,0.0,-1.0,0.0,0
+
+	fx_exhaustFlame_white_tiny = 0.0, -.12673, 0.0, 0.0, 1.0, 0.0, running
+
+	sound_vent_soft = engage
+	sound_rocket_soft = running
+	sound_vent_soft = disengage
+	sound_explosion_low = flameout
+
+	TechRequired = start
+	entryCost = 0
+	cost = 1
+	category = Engine
+	subcategory = 0
+	title = Pioneer Adjustment Motor
+	manufacturer = STL
+	description = TBD
+
+	attachRules = 1,0,1,1,0
+
+	dragModelType = default
+	maximum_drag = 0.01
+	minimum_drag = 0.005
+	angularDrag = 2
+	crashTolerance = 10
+	breakingForce = 10000
+	breakingTorque = 10000
+	stagingIcon = SOLID_BOOSTER
+	
+	skinTempTag = Steel
+	internalTempTag = Steel
+	
+	engineType = PioneerVernier
+	
+	MODULE
+	{
+		name = ModuleEnginesRF
+		thrustVectorTransformName = thrustTransform
+		
+		EngineType = SolidBooster
+		allowShutdown = False
+		throttleLocked = True
+	}
+	
+	MODULE
+	{
+		name = FXModuleAnimateThrottle
+		animationName = SRBs
+		responseSpeed = 0.001
+		dependOnEngineState = True
+		dependOnThrottle = True
+	}
+}
+
+
+@PART[PioneerVernier]:FOR[RealismOverhaul]:NEEDS[TextureReplacer,!TexturesUnlimited]
+{
+	MODULE
+	{
+		name = TRReflection
+		shader = Reflective/VertexLit
+		meshes = core, base
+		color = 0.5 0.5 0.5
+		interval = 2
+	}
+}
+
+KSP_TEXTURE_SET:NEEDS[TexturesUnlimited]
+{
+	name = RO-PioneerVernier
+
+	MATERIAL
+	{
+		shader = TU/Metallic
+		keyword = TU_STOCK_SPEC
+
+		texture = _MainTex, ROTanks/Assets/Textures/Metal-DIFF
+		texture = _BumpMap, ROTanks/Assets/Textures/Smooth-NRM
+		
+		PROPERTY
+		{
+			name = _Color
+			// Higher numbers are lighter
+			color = 1.8, 1.8, 1.8
+		}
+		PROPERTY
+		{
+			name = _Metal
+			// 0 = not metallic, 1 = mirror finish
+			float = 1
+		}
+		PROPERTY
+		{
+			name = _Smoothness
+			// 0 = most blurry, 1 = not blurry
+			float = 0.9
+		}
+	}
+}
+
+KSP_TEXTURE_SET:NEEDS[TexturesUnlimited] {
+	name = RO-PioneerVernier_Recolor
+	recolorable = true
+	
+	MATERIAL
+    {
+        shader = TU/Metallic
+        texture = _MainTex, ROTanks/Assets/Textures/Metal-DIFF
+        texture = _BumpMap, ROTanks/Assets/Textures/Smooth-NRM
+        texture = _MetallicGlossMap, ROTanks/Assets/Textures/Smooth-METAL
+        texture = _MaskTex, ROEngines/Assets/ItsRyan/PioneerVernier-MASK
```

Comment:
This is probably not advisable; can you just make a copy of them here?

Reply:
Yes, I'll change that.  Just was thinking I didn't want to claim ownership of those textures, but it does create a dependency.